sycl/source/detail/scheduler/scheduler.cpp

```diff
@@ -9,14 +9,11 @@
 #include "detail/sycl_mem_obj_i.hpp"
```

Comment:
NIT, maybe at the same time you could align the system vs local include, eve though it's not touched in this diff?